.gitlab-ci.yml

```diff
@@ -311,22 +311,25 @@ run_tests_rpm-x64-py3:
 run_tests_ebpf:
   stage: source_test
   needs: ["fail_on_non_triggered_tag", "build_libbcc_x64"]
-  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/system-probe_x64:$DATADOG_AGENT_BUILDIMAGES
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/system-probe_x64:$DATADOG_AGENT_SYSPROBE_BUILDIMAGES
   before_script:
     - mkdir -p $CI_PROJECT_DIR/.tmp/binary-ebpf
     - cd $SRC_PATH
     - pip3 install distro -c requirements.txt # needed for some of our invoke tasks until the builders image contain it
-    - inv -e deps --verbose --dep-vendor-only
+    - apt-get install -y curl
+    - eval "$(gimme $SYSTEM_PROBE_GO_VERSION)"
+    - inv -e deps --verbose --no-checks
```

Comment:
Once this is merged, I can propagate some of this changes to the build image and clean up this.